README.md

```diff
@@ -0,0 +1,24 @@
+This is a boilerplate extension for experiments. It *is not* designed to work out of the box. It *is* designed to be:
+* something you can clone from this repository
+* alter some key values
+* create a simple experiment that you can use elsewhere
+
+To use (on Linux or OS X):
+
+```
+git clone https://github.com/web-ext-experiments/boilerplate-experiment
```

Comment:
github has a thing where you can download a tarball with a particular revision rather than cloning the whole repo, something like:
```
curl -L https://apis.github.com/repos/web-ext-experiments/boilerplate-experiment | tar xf -
```
But given that this repo isn't likely to get big or have lots of history, maybe that's not worth it here.